website/docs/docs/bi-directional-contract-testing/compatibility-checks.md

```diff
@@ -276,30 +275,30 @@ Error paths use a JSONPath-like syntax with an associated value, to help you nav
 
 ##### `additionalProperties`
 
-When a pact test has an expectation for a response body, it may ask for a subset of what the provider can provide - this is perfectly acceptable. However it cannot ask for a property not present in the spec - this will cause a failure - and is the most common error of this kind.
+When a pact test expectats a response body, it may ask for a subset of what the provider can provide - this is perfectly acceptable. However it cannot ask for a property not present in the spec - this will cause failure - and is the most common error of this kind.
```

Comment:
```suggestion
When a pact test expects a response body, it may ask for a subset of what the provider can provide - this is perfectly acceptable. However it cannot ask for a property not present in the spec - this will cause failure - and is the most common error of this kind.
```